app/Model/Database/Advanced/Entity/Article.php

```diff
@@ -39,6 +39,7 @@ class Article implements Translatable
 	/**
 	 * @Gedmo\Slug(fields={"title"})
 	 * @ORM\Column(length=128, unique=true)
+	 * @phpstan-ignore-next-line
```

Comment:
Proc se to ignoruje?

Reply:
PHPStan házel chybu

Reply:
Jakou?

Reply:
```
  66     Property App\Model\Database\Advanced\Entity\Article::$created is never written, only read.  
         💡 See: https://phpstan.org/developing-extensions/always-read-written-properties             
```

Reply:
vyřešeno dle https://github.com/phpstan/phpstan-doctrine/issues/231#issuecomment-965268968